zookeeper-server/src/main/java/org/apache/zookeeper/AsyncCallback.java

```diff
@@ -41,46 +46,45 @@
 
         /**
          * Process the result of the asynchronous call.
-         * <p>
-         * On success, rc is
-         * {@link org.apache.zookeeper.KeeperException.Code#OK}.
-         * <p>
-         * On failure, rc is set to the corresponding failure code in
-         * {@link org.apache.zookeeper.KeeperException}.
+         *
+         * <p>On success, rc is {@link KeeperException.Code#OK}.
+         *
+         * <p>On failure, rc is set to the corresponding failure code in {@link KeeperException}.
          * <ul>
-         * <li>
-         * {@link org.apache.zookeeper.KeeperException.Code#NONODE}
-         * - The node on given path doesn't exist for some API calls.
-         * </li>
-         * <li>
-         * {@link org.apache.zookeeper.KeeperException.Code#BADVERSION}
-         * - The given version doesn't match the node's version
-         * for some API calls.
-         * </li>
+         *  <li> {@link KeeperException.Code#NONODE}
+         *              - The node on given path doesn't exist for some API calls.</li>
+         *  <li> {@link KeeperException.Code#BADVERSION}
+         *              - The given version doesn't match the node's version for some API calls.</li>
          * </ul>
          *
          * @param rc   The return code or the result of the call.
          * @param path The path that we passed to asynchronous calls.
-         * @param ctx  Whatever context object that we passed to
-         *             asynchronous calls.
-         * @param stat {@link org.apache.zookeeper.data.Stat} object of
-         *             the node on given path.
+         * @param ctx  Whatever context object that we passed to asynchronous calls.
+         * @param stat {@link Stat} object of the node on given path.
+         *
+         * @see ZooKeeper#exists(String, boolean, StatCallback, Object)
+         * @see ZooKeeper#exists(String, Watcher, StatCallback, Object)
+         * @see ZooKeeper#setData(String, byte[], int, StatCallback, Object)
+         * @see ZooKeeper#setACL(String, List, int, StatCallback, Object)
          */
         void processResult(int rc, String path, Object ctx, Stat stat);
 
     }
 
     /**
-     *  This callback is used to get all children node number of the node.
+     * This callback is used to get all children node number of the node.
+     *
+     * @since 3.6.0
      */
     @InterfaceAudience.Public
     interface AllChildrenNumberCallback extends AsyncCallback {
 
         /**
-         *
          * @param rc      The return code or the result of the call.
          * @param ctx     Whatever context object that we passed to asynchronous calls.
-         * @param number  the number of children nodes under a specific path
+         * @param number  The number of children nodes under a specific path
```

Comment:
nit - missing full stop (should be "path.")

Reply:
Thanks!